spotbugs/src/main/java/edu/umd/cs/findbugs/detect/CbeckMustOverrideSuperAnnotation.java

```diff
@@ -69,11 +61,22 @@ public void visit(Code code) {
         sawCallToSuper = false;
         super.visit(code);
         if (!sawCallToSuper) {
-            bugReporter.reportBug(new BugInstance(this, "TESTING", NORMAL_PRIORITY).addClassAndMethod(this).addString(
+            bugReporter.reportBug(new BugInstance(this, "OVERRIDING_METHODS_MUST_INVOKE_SUPER", NORMAL_PRIORITY).addClassAndMethod(this).addString(
                     "Method must invoke override method in superclass"));
         }
     }
 
+    public String getEffectiveMethodSig() {
+        String methodSig = getMethodSig();
```

Comment:
No point calling `getMethodSig` if we have a bridge. Can this call be deferred until we check that?

Reply:
OK, I'll change it.